web/app/services/build-log-websocket.service.ts

```diff
@@ -36,7 +36,7 @@ export class BuildLogWebsocketService {
   }
 
   private createSocketUrl(projectId: string, buildNumber: number) {
-    return `${this.API_ROOT}/projects/${projectId}/builds/${
-        buildNumber}/log.ws`;
+    return `${this.API_ROOT}?project_id=${projectId}&build_number=${
```

Comment:
Oh, do we have access control over this? As in, does this api ensure that the caller actually has permission to connect and stream the output of this build?

Reply:
I don't think we have any of that yet. This is the legacy method of building. Just getting placeholders here in place until the new agent work is complete.

Reply:
🆒